app/components/content-icon.js

```diff
@@ -4,7 +4,7 @@ export default Ember.Component.extend({
   tagName: 'img',
   classNames: ['icon'],
   attributeBindings: ['src'],
-  src: Ember.computed('contentType', function() {
-    return `/assets/${this.get('contentType')}-icon.svg`;
+  src: Ember.computed('icon-image', function() {
```

Comment:
I think camelCase for attribute names is convention. This should probably be changed for `help-text` and `radio-name` as well.